client/src/utils/jobsUtil.js

```diff
@@ -61,69 +71,85 @@ export function getJobDisplayInformation(job, app) {
     // Improve any values with app information
     try {
       try {
-        display.systemName = getSystemName(app.exec_sys.login.host);
+        display.systemName = getSystemName(app.exec_sys.host);
       } catch (ignore) {
         // ignore if there is problem improving the system name
       }
 
-      display.applicationName = app.definition.label;
+      // TODO V3: Undefined for now
+      // display.applicationName = app.definition.notes.label;
```

Comment:
For now, let's just show the `name` of the application here

Reply:
This would be `appId`, right? (instead of the job `name`)
This is set by default above in creating the display object.
I'll either comment or remove this section for now.
https://github.com/TACC/Core-Portal/blob/3a5cc174a72c704c38617c8056d8e36cbbb4b9c4/client/src/utils/jobsUtil.js#L50-L53

Reply:
@duckonomy Yes, `appId`. Confused myself with how `appId` replaced `name` in v3.

> This is set by default above in creating the display object.
I'll either comment or remove this section for now.

Sounds great 👍